docs/_docs/testnet/faucet.md

```diff
@@ -7,19 +7,22 @@ Using the steps below you will request testnet OBX tokens from the faucet availa
 * Your wallet address.
 * Access to the [Obscuro Discord server](https://discord.gg/yQfmKeNzNd).
 * (Optional) [MetaMask](https://metamask.io/) wallet installed in your browser.
-* (Optional) A local copy of the [Obscuro MetaMask wallet extension](../wallet-extension/wallet-extension.md).
+* (Optional) A local copy of the [Obscuro MetaMask wallet extension](https://docs.obscu.ro/wallet-extension/wallet-extension/).
 
 ## Requesting Testnet OBX Tokens
 1. Make a note of your wallet address or copy it to your clipboard.
-1. Open the [_faucet-requests_ channel](https://discord.gg/5qyj3qraaH) on Obscuro Discord.
-1. Request OBX using the `/faucet` command. The faucet will credit 100,000 OBX by default:
-![faucet command](../assets/images/faucet-cmd.png)
-1. Provide your wallet address and hit Enter. The faucet will acknowledge your request:
-![faucet ack](../assets/images/faucet-ack.png)
-1. After a short period of time the faucet will confirm the Testnet OBX tokens have been credited to your wallet:
-![faucet complete](../assets/images/faucet-done.png)
+2. Open the [_faucet-requests_ channel](https://discord.gg/5qyj3qraaH) on Obscuro Discord.
```

Comment:
@GavT the discord link seems to point me to the welcome channel? It might just be the way discord works though

Reply:
@moraygrieve that is the standard Discord behaviour if you are not already logged in using a browser

Reply:
Got it - have approved.